### PR TITLE
Add tests for reviewers list

### PR DESCRIPTION
**Pull request by TimvdLippe**
Small test suite to verify the ordering.

### CODE REVIEW

File: test/projects/pull-request/reviewers-list.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      elem.reviewers = [{lgtm: true, avatar_url: 'resources/one.jpg'}, {lgtm: false, avatar_url: 'resources/two.jpg'}];
+      flush(function() {
+        var reviewers = getReviewers();
+        assert.include(getComputedStyle(reviewers[0])['background-image'], 'resources/one.jpg');
+        assert.include(getComputedStyle(reviewers[1])['background-image'], 'resources/two.jpg');
+        done();
+      });
+    });
+
+    test('reviewers are sorted by lgtm', function(done) {
```
Comment (eanker) [inline test/projects/pull-request/reviewers-list.html:43]: Is this true?
You test with includes, where the order doesn't matter.

Comment (TimvdLippe) [inline test/projects/pull-request/reviewers-list.html:43]: Include is performed on the background-image of the reviewers list. This is from the element list, so the order is asserted.

Comment (eanker) [inline test/projects/pull-request/reviewers-list.html:43]: Thanks, it's resolved now.

**Comment by eanker** [comment]
lgtm

**Pull request merged**